[TOOL] pidcat  (by @appknox) (#2895)
OWASP/owasp-mastg: tools/android/MASTG-TOOL-0108.md

---
title: pidcat
platform: android
source: https://github.com/JakeWharton/pidcat
---

The [Pidcat](https://github.com/JakeWharton/pidcat) tool acts as a wrapper for `adb logcat` that allows you to view Android log entries for specific application packages in a visually distinct color format.

<img src="https://raw.githubusercontent.com/JakeWharton/pidcat/master/screen.png"/>

> 

Reviewer: ```suggestion
```

---
title: pidcat
platform: android
source: https://github.com/JakeWharton/pidcat
---

> The [Pidcat](https://github.com/JakeWharton/pidcat) tool acts as a wrapper for adb logcat and includes an update to Jeff Sharkey's remarkable [logcat color script](https://jsharkey.org/logcat/). It facilitates the viewing of Android log entries for specific application packages in a visually distinct color format. This script effectively filters logs by application package, allowing us to input the target package as the only argument to the python script.

Reviewer: ```suggestion
The [Pidcat](https://github.com/JakeWharton/pidcat) tool acts as a wrapper for `adb logcat` that allows you to view Android log entries for specific application packages in a visually distinct color format.

<img src="https://raw.githubusercontent.com/JakeWharton/pidcat/master/screen.png"/>

```
